src/test/java/com/zebrunner/carina/demo/MobileTestLuna.java

```diff
@@ -0,0 +1,125 @@
+        }
+    }
+
+    @Test()
+    @MethodOwner(owner = "jaco")
+    @TestLabel(name = "feature", value = {"mobile", "regression"})
+    public void searchTest() {
+        initPage(getDriver(), WelcomePageBase.class);
+        ApptoidePresentation apptoidePresentation = new ApptoidePresentation(getDriver());
+        ApptoideBase apptoideBase = apptoidePresentation.clickSkipBtn();
+        String searchValue = "Pixel Dungeon";
+
+        List<WebElement> apps = apptoideBase.search(searchValue);
+
+        WebElement app = apps.stream()
+                    .filter(result ->
+                            result.findElement(By.xpath("//*[@resource-id='cm.aptoide.pt:id/app_name']")).getText().equals(searchValue))
+                    .findFirst()
+                    .orElse(null);
+
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertNotEquals(app, null, "search value doesn't exist");
+
+        apps.getFirst().click();
+        ApptoideAppPage appPage = new ApptoideAppPage(getDriver());
+        for (String s : StringUtils.split(searchValue, " ")) {
+            softAssert.assertTrue(appPage.getName().contains(s), "the app doesnt have the name of the search on the title");
+        }
+        softAssert.assertAll();
+    }
+
+    @Test()
+    @MethodOwner(owner = "jaco")
+    @TestLabel(name = "feature", value = {"mobile", "regression"})
+    public void testToolBar() {
+        initPage(getDriver(), WelcomePageBase.class);
+        ApptoidePresentation apptoidePresentation = new ApptoidePresentation(getDriver());
+        ApptoideBase apptoideBase = apptoidePresentation.clickSkipBtn();
+        for (TabValue value : TabValue.values()) {
+            if (!value.equals(TabValue.SEARCH))
+                Assert.assertNotNull(apptoideBase.selectPage(value).toolBar, value.getValue() + " doesnt have the tool Bar");
+        }
+    }
+
+    @Test()
+    @MethodOwner(owner = "jaco")
+    @TestLabel(name = "feature", value = {"mobile", "regression"})
+    public void testHome() {
+        initPage(getDriver(), WelcomePageBase.class);
+        ApptoidePresentation apptoidePresentation = new ApptoidePresentation(getDriver());
+        ApptoideBase apptoideBase = apptoidePresentation.clickSkipBtn();
+
+        for (Chip chip : Chip.values()) {
+            ((ApptoideHome)apptoideBase.selectPage(TabValue.HOME)).clickChip(chip);
+        }
+    }
+
+    @Test()
+    @MethodOwner(owner = "jaco")
+    @TestLabel(name = "feature", value = {"mobile", "regression"})
+    public void testStore() {
+        initPage(getDriver(), WelcomePageBase.class);
+        ApptoidePresentation apptoidePresentation = new ApptoidePresentation(getDriver());
+        ApptoideBase apptoideBase = apptoidePresentation.clickSkipBtn();
+
+        ApptoideStores apptoideStores = (ApptoideStores)apptoideBase.selectPage(TabValue.STORES);
+        List<WebElement> followedStores = apptoideStores.getFollowedStores();
```

Comment:
The same thing, here it was possible to return the followedStores size from the page and then use this size in the check.